server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -114,6 +121,19 @@ public DefaultConnectOptionsContext(ServiceConnectOptionsType connectOptions, Se
                 }
             }
 
+            ServiceConnectOptionsType.HttpKeepalive.Enum alive = connectOptions.getHttpKeepalive();
+            if (alive != null) {
+                tmpHttpKeepaliveEnabled = alive != ServiceConnectOptionsType.HttpKeepalive.DISABLED;
+            }
+
+            String timeoutValue = connectOptions.getHttpKeepaliveTimeout();
+            if (timeoutValue != null) {
+                long val = Utils.parseTimeInterval(timeoutValue, TimeUnit.SECONDS);
+                if (val > 0) {
+                    tmpHttpKeepaliveTimeout = val;
+                }
+            }
+
         }
 
         // Set default values via service-defaults
```

Comment:
The schema says "This is the element for configuring a set of default accept-options for all services running on the gateway." Are there service-defaults for connect-options also ?